docs/application/dotnet/guides/multimedia/thumbnail-images.md

```diff
@@ -27,17 +27,43 @@ using Tizen.Multimedia.Util;
 > **Note**   
 > The input media file can be common content in the device storage (internal or external) or private content in your application data.
 
-<a name="get_thumbnail"></a>
+
 ## Extracting a Thumbnail
 
 To extract a thumbnail from a file, use the `ExtractAsync()` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:
 
 ```
-async Task Extract(string testImagePath)
+async Task ExtractAsync(string testImagePath, Size size)
+{
+    ThumbnailExtractionResult result = await ThumbnailExtractor.ExtractAsync(testImagePath, size);
+
+    Log.Info("Util", "Data size is " + result.RawData.Length);
+}
+```
+
+## Extracting a Thumbnail synchronously
```

Comment:
@hsgwon Slight change needed.

Current :
## Extracting a Thumbnail synchronously
Change to:
## Extracting a Thumbnail Synchronously

Reason: In Tizen Docs,  we follow title case style fo all headings. The first letter of each word must be a capital letter.

Reply:
OK I'll fix it.